File: 1_class_object.cpp
#include <bits/stdc++.h>
using namespace std;
class ClassObject {
public:
    // Constructor
    ClassObject() {
        cout << "Constructor" << endl;
    }
    // Method
    void display() {
        cout << "Method" << endl;
    }
    // Destructor
    ~ClassObject() {
        cout << "Destructor" << endl;
    }

};

int main() {
    ClassObject obj;
    obj.display();
    return 0;
}

File: 1_class_object.py
class classObject:

    def __init__(self) -> None:
        print("Constructor")
    
    def display(self):
        print("Method")

    def __del__(self):
        print("Destructor")

    def __str__(self) -> str:
        return "Class"

obj = classObject()
obj.display()
print(obj)

